rotkehlchen/chain/ethereum/modules/airdrops/decoder.py

```diff
@@ -57,6 +68,9 @@
 ELFI_LOCKING = string_to_evm_address('0x02Bd4A3b1b95b01F2Aa61655415A5d3EAAcaafdD')
 ELFI_VOTE_CHANGE = b'3\x16\x1c\xf2\xda(\xd7G\xbe\x9d\xf16\xb6\xf3r\x93\x90)\x84\x94\x94rht1\x93\xc5=s\xd3\xc2\xe0'  # noqa: E501
 
+COW_PROTOCOL = string_to_evm_address('0xD057B63f5E69CF1B929b356b579Cba08D7688048')
+COW_PROTOCOL_CLAIMED = b'\xd46\xe9\x97=\x1eD\xd4\r\xb4\xd4\x11\x9e<w<\xad\xb12;&9\x81\x96\x8c\x14\xd3\xd1\x91\xc0\xe1H'  # noqa: E501
+
```

Comment:
this here is not used in this file for what I'm seeing and is a duplicate. I believe you forgot to remove it